src/modules/Elsa.EntityFrameworkCore.MySql/Migrations/Alterations/20240305212213_V3_1.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Elsa.EntityFrameworkCore.MySql.Migrations.Alterations
+{
+    /// <inheritdoc />
+    public partial class V3_1 : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                name: "Status",
+                schema: "Elsa",
```

Comment:
Add custom schema as the Initials migrations

Reply:
Hi, thanks for the reminder. We will do this before the 3.1 release. Given that it’s a manual process and we might face more migration updates, it’s more efficient to do it once rather than multiple times.